resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -202,6 +206,10 @@
     fpm_opts << "--directories #{dir}"
   end
 
+  options.rpm_triggers.each do |trigger|
+    fpm_opts << "--rpm-trigger-after-install #{trigger}"
+  end
+
```

Comment:
Is there a distinction between install and upgrade triggers? You have separate keys for them, but I don't see how they're being treated differently.

Reply:
They're treated differently in the script generation at https://github.com/sarameisburger/ezbake/blob/9f8404dd4e1cc917c8b47b2e7d86704eaa39ea6a/resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb#L133-L153

Reply:
@underscorgan afaict, those are doing the exact same thing, just with different sets of triggers.. Am I missing something?

Reply:
good catch, see comment in controller.sh.erb

Reply:
Is there a reason for not going straight to
```
opts.on('--rpm-trigger TRIGGER', 'TRIGGER for the rpm packages, in the format package:file_containing_script') do |t|
  fpm_opts << "--rpm-trigger-after-install #{t}"
end
```
?

Reply:
We only want to be adding those options if our package type is RPM, which we don't know in the option parser.

Reply:
mmmkay